testcases/OpTestInbandIPMI.py

```diff
@@ -152,13 +73,25 @@ def run_ipmi_cmd_on_host(self, i_cmd):
             raise OpTestError(l_msg)
```

Comment:
Should be fixed by b3269f622d5b, which refactors it to use the new run_command code as well as use a custom exception rather than generic OpTestError